src/features/activityId/components/map/kakao-map.tsx

```
@@ -2,16 +2,18 @@
 
 import Image from 'next/image';
 import { useEffect, useState } from 'react';
-import { Map, MapMarker, useKakaoLoader } from 'react-kakao-maps-sdk';
+import { CustomOverlayMap, Map, useKakaoLoader } from 'react-kakao-maps-sdk';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)